data/moves.js

```diff
@@ -16339,8 +16351,8 @@ exports.BattleMovedex = {
 		onHit: function (target, source) {
 			if (target.boosts.atk === -6) return false;
 			let atk = target.getStat('atk', false, true);
-			this.boost({atk:-1}, target, source, null, null, true);
-			this.heal(atk, source, target);
+			let didSomething = this.boost({atk:-1}, target, source, null, null, true);
+			return this.heal(atk, source, target) || didSomething;
```

Comment:
Is this the actual Strength Sap mechanic?

Reply:
It only now occurred to me that I used it instead of and

Reply:
No wait “or” is right, I’m just second-guessing myself

Reply:
This is definitely not documented in the Strength Sap description. @Marty-D?

Reply:
I assumed that like almost every other Status move, "this move fails if it accomplishes nothing" would have been obvious.